WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/db/WatchFaceDatabaseCallback.kt

```diff
@@ -45,9 +45,15 @@ private const val MINUTE_HAND_WIDTH_FRACTION = 0.0163f
 private const val SECOND_HAND_LENGTH_FRACTION = 0.37383f
 private const val SECOND_HAND_WIDTH_FRACTION = 0.00934f
 
+// Used for corner roundness of the arms.
+private const val ROUNDED_RECTANGLE_CORNERS_RADIUS = 1.5f
+private const val SQUARE_RECTANGLE_CORNERS_RADIUS = 0.0f
+
 // Primary key ids assigned for various style and dimension elements in the database.
 // These are accessible to the watch face, so it can assign the element the user chooses in
 // the watch face settings.
+const val analogWatchFaceKeyId = 1
```

Comment:
i think this is only used in this file and can be made private

Reply:
I use it in my next CL. The analogWatch face surface needs to know the key that matches it.